app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -85,7 +85,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_openNavigationDrawerAndClose_closingOfNavigationDrawerIsVerifiedSuccessfully() {
+  fun openHelpActivity_openNavigationDrawerAndClose_closingOfNavigationDrawerIsVerifiedSuccessfully() { // ktlint-disable max-line-length
```

Comment:
Ditto.

Reply:
We cannot do that because ktlint will check that line only in which this comment is written which is more than 100 char